File: notify/v1/binding/create-binding-server-fcm/create-binding-server-fcm.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/console
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$binding = $twilio->notify->v1->services("ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
                              ->bindings
                              ->create("00000001",
                                       "fcm",
                                       "fcm_device_token",
                                       array(
                                           "endpoint" => "XXXXXXXXXXXXXXX",
                                           "tag" => "preferred device"
                                       )
                              );

print($binding->sid);

File: notify/v1/binding/create-binding-server-fcm/create-binding-server-fcm.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Converters;
using Twilio.Rest.Notify.V1.Service;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var binding = BindingResource.Create(
            endpoint: "XXXXXXXXXXXXXXX",
            tag: Promoter.ListOfOne("preferred device"),
            identity: "00000001",
            bindingType: BindingResource.BindingTypeEnum.Fcm,
            address: "fcm_device_token",
            pathServiceSid: "ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
        );

        Console.WriteLine(binding.Sid);
    }
}
